wallet/wallet.go

```diff
@@ -3706,6 +3713,79 @@ func (w *Wallet) LockedOutpoints() []dcrdtypes.TransactionInput {
 	return locked
 }
 
+func (w *Wallet) acctLockedOutpoints(ctx context.Context, acctName string) ([]dcrdtypes.TransactionInput, error) {
+	w.lockedOutpointMu.Lock()
+	locked := make([]wire.OutPoint, len(w.lockedOutpoints))
+	i := 0
+	for op := range w.lockedOutpoints {
+		locked[i] = op
+		i++
+	}
+	w.lockedOutpointMu.Unlock()
+
+	acctLocked := make([]dcrdtypes.TransactionInput, 0, len(w.lockedOutpoints))
+	err := walletdb.View(ctx, w.db, func(tx walletdb.ReadTx) error {
+		addrmgrNs := tx.ReadBucket(waddrmgrNamespaceKey)
+		txmgrNs := tx.ReadBucket(wtxmgrNamespaceKey)
+
+		defaultAccountName, err := w.manager.AccountName(
+			addrmgrNs, udb.DefaultAccountNum)
+		if err != nil {
+			return err
+		}
+
+		// Outpoints are not validated before they're locked, so
+		// invalid outpoints may be locked. Simply ignore.
+		for _, op := range locked {
```

Comment:
The slice isn't of pointers so `op` is 37 bytes.  I'd consider `op := &locked[i]` as the first line in the loop instead, but it's a minor thing.